server/Arcadia.Assistant.CSP/Vacations/EmployeesQueryExecutor.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Arcadia.Assistant.CSP.Vacations
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    using Microsoft.EntityFrameworkCore;
+
+    using Arcadia.Assistant.CSP.Model;
+
+    public class EmployeesQueryExecutor
```

Comment:
why is this class needed?

Reply:
To not have loading of employees logic in vacations registry itself.
There are 3 parts:
- employees executor which knows how to get all employees by some conditions
- vacations email loader that periodically pushes new vacations information
- vacations registry that combines them together